### PR TITLE
DAT-3993 | Change template title while editing infobox template

### PR DESCRIPTION
**Pull request by dianafa**
## Links

* https://wikia-inc.atlassian.net/browse/DAT-3993

## Description

User can change template title while editing infobox template

## Reviewers

@RafLeszczynski @Warkot

### CODE REVIEW

File: front/main/app/routes/infobox-builder.js [hunk 1]
```diff
@@ -42,7 +42,10 @@ export default Ember.Route.extend(ConfirmationMixin, {
 	 * @returns {Object}
 	 */
 	model(params) {
-		return InfoboxBuilderModel.create({title: params.templateName});
+		return InfoboxBuilderModel.create({
+			title: params.templateName,
+			initialTitle: params.templateName
```
Comment (RafLeszczynski) [inline front/main/app/routes/infobox-builder.js]: I wander if we should keep this in model as it's not connected directly with model data which is saved to server. What do you think about moving this to controller?

File: front/main/app/components/infobox-builder.js [hunk 2]
```diff
@@ -296,6 +305,23 @@ export default Ember.Component.extend(
 		},
 
 		/**
+		 * Set new title, hide modal and call callback function if set.
+		 *
+		 * @param {String} title new title
```
Comment (tomasznapieralski) [inline front/main/app/components/infobox-builder.js]: don't you think that param name suggests it's purpose? :)

File: front/main/tests/unit/components/infobox-builder-test.js [hunk 3]
```diff
@@ -507,3 +507,16 @@ test('correctly calculates showEditTitleModalCancelButton', function (assert) {
 		assert.equal(component.get('showEditTitleModalCancelButton'), testCase.showEditTitleModalCancelButton);
 	});
 });
+
+
```
Comment (tomasznapieralski) [inline front/main/tests/unit/components/infobox-builder-test.js]: remove this line, please

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by RafLeszczynski** [comment]
LGTM

**Pull request merged**